Fix deprecated Type method call in elasticsearch

```bash
golangci-lint run
common/elastic/client.go:41:17: SA1019: c.client.Index().
	Index(index).
	Type is deprecated: Types are in the process of being removed. (staticcheck)
	result, err := c.client.Index().
	               ^
common/elastic/client.go:56:17: SA1019: c.client.Index().
	Index(index).
	Id(id).
	Type is deprecated: Types are in the process of being removed. (staticcheck)
	result, err := c.client.Index().
	               ^
common/elastic/client.go:95:12: SA1019: c.client.Update().
	Index(index).
	Type is deprecated: Types are in the process of being removed. (staticcheck)
	_, err := c.client.Update().
	          ^
common/elastic/client.go:114:12: SA1019: c.client.Delete().
	Index(index).
	Type is deprecated: Types are in the process of being removed. (staticcheck)
	_, err := c.client.Delete().
	          ^
```

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -59,10 +59,3 @@ linters-settings:
     # 2. Third-party packages
     # 3. Packages from the same project
     local-prefixes: github.com/MurmurationsNetwork/MurmurationsServices
-
-issues:
-  exclude-rules:
-    - linters:
-        - staticcheck
-      # text: Ignore using a deprecated function, variable, constant or field.
-      text: "SA1019"

--- a/pkg/elastic/client.go
+++ b/pkg/elastic/client.go
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	elastic "github.com/olivere/elastic/v7"
+	"github.com/olivere/elastic/v7"
 
 	"github.com/MurmurationsNetwork/MurmurationsServices/pkg/logger"
 )
@@ -46,7 +46,6 @@ func (c *esClient) Index(
 	ctx := context.Background()
 	result, err := c.client.Index().
 		Index(index).
-		Type(docType).
 		BodyJson(doc).
 		Do(ctx)
 	if err != nil {
@@ -72,7 +71,6 @@ func (c *esClient) IndexWithID(
 	result, err := c.client.Index().
 		Index(index).
 		Id(id).
-		Type("_doc").
 		BodyJson(doc).
 		Do(ctx)
 	if err != nil {
@@ -129,7 +127,6 @@ func (c *esClient) Update(
 	ctx := context.Background()
 	_, err := c.client.Update().
 		Index(index).
-		Type(docType).
 		Id(id).
 		Doc(update).
 		Do(ctx)
@@ -154,7 +151,6 @@ func (c *esClient) Delete(index string, id string) error {
 	ctx := context.Background()
 	_, err := c.client.Delete().
 		Index(index).
-		Type(docType).
 		Id(id).
 		Do(ctx)
 	if err != nil {

--- a/pkg/elastic/constant.go
+++ b/pkg/elastic/constant.go
@@ -1,3 +1,0 @@
-package elastic
-
-const docType = "_doc"

--- a/pkg/mongo/client.go
+++ b/pkg/mongo/client.go
@@ -36,10 +36,8 @@ func (c *mongoClient) Count(
 
 func (c *mongoClient) InsertOne(collection string, document interface{},
 	opts ...*options.InsertOneOptions) (*mongo.InsertOneResult, error) {
-	mergedOpt := options.MergeInsertOneOptions(opts...)
-
 	result, err := c.db.Collection(collection).
-		InsertOne(context.Background(), document, mergedOpt)
+		InsertOne(context.Background(), document, opts...)
 	if err != nil {
 		return nil, err
 	}
@@ -57,13 +55,12 @@ func (c *mongoClient) FindOneAndUpdate(
 		opts,
 		options.FindOneAndUpdate().SetReturnDocument(options.After),
 	)
-	mergedOpt := options.MergeFindOneAndUpdateOptions(opts...)
 
 	// Automatically increment the document version.
 	update["$inc"] = bson.M{"__v": 1}
 
 	result := c.db.Collection(collection).
-		FindOneAndUpdate(context.Background(), filter, update, mergedOpt)
+		FindOneAndUpdate(context.Background(), filter, update, opts...)
 	if result.Err() != nil {
 		return nil, result.Err()
 	}

